<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRequest</class>
 <widget class="QDialog" name="CreateRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="war2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>72</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>26</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>id станка: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_id"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>email: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_email"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Номер телефона: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_phone"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Имя: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Фамилия: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_surname"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Отчество: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_patronymic"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Описание: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_message"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="war3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>116</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="war7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>286</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="war6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="war5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>198</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>371</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Создание</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusBar">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>426</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="_broken_list">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>376</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>316</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Вводите список сломанных деталей в формате деталь|состояние:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>356</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Состояние (Рабочий = 0, В ремонте = 1, Сломан = 2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="war4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>156</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>336</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Деталь (Ремень = 0, Двигатель = 1)</string>
   </property>
  </widget>
  <widget class="QLabel" name="war1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
